src/components/molecules/ProductSlider/ProductCard.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+import styled from 'styled-components'
+
+// C O M P O N E N T S
+import { Button } from '../../atoms/Button/Button'
+import { Card } from '../Card/Card'
+import { Icon } from '../../atoms/Icon/Icon'
+import { Label } from '../../atoms/Label/Label'
+
+export interface ProductCardProps {
```

Comment:
It would be nice that we could give a `className` and a `style` props to this component:

```suggestion
export interface ProductCardProps extends DefaultComponentProps {
```

... and consume it like:

```tsx
    <StyledProductCard className={className} style={style}>
```